doc/api/packages.md

```diff
@@ -211,10 +212,9 @@ To set the main entry point for a package, it is advisable to define both
 }
 ```
 
-The benefit of doing this is that when using the [`"exports"`][] field all
-subpaths of the package will no longer be available to importers under
-`require('pkg/subpath.js')`, and instead they will get a new error,
-`ERR_PACKAGE_PATH_NOT_EXPORTED`.
+When defining the [`"exports"`][] field all subpaths of the package will be
```

Comment:
nit: I would put a comma here to improve readability, feel free to disregard though.

```suggestion
When defining the [`"exports"`][] field, all subpaths of the package will be
```